javascript/extractor/src/com/semmle/jcorn/ESNextParser.java

```diff
@@ -413,6 +416,20 @@ protected Statement parseImport(Position startLoc) {
     }
   }
 
+  /**
+   * Parses an import.meta expression, assuming that the initial "import" has been consumed.
+   */
+  private MetaProperty parseImportMeta(Position loc) {
+    Identifier meta = new Identifier(new SourceLocation(loc), "import");
```

Comment:
Not sure "meta" is a great name; it's the property that's `meta`, not the `import`.

Reply:
It follows the naming in `MetaProperty`.  
The "meta"-object is `import`, and the property on the "meta"-object is `meta`. 

But yes, its an unfortunate clash of names, so I'm renaming it to `importIdentifier`.

Reply:
Actually no. I'll inline the creation of the identifier.